mnemonic-core/src/test/java/org/apache/mnemonic/DurablePersonNGTest.java

```diff
@@ -78,25 +78,54 @@ public boolean reclaim(Long mres, Long sz) {
         }
 
         person = PersonFactory.create(act);
+        int size = rand.nextInt(1024 * 1024) + 1024 * 1024;
+/////////////////////////if size is smaller than remaining act capacity////
+
+        mbh = act.createBuffer(size);
+        if (mbh != null) {
+          Assert.assertNotNull(mbh);
+          for (int i = 0; i < size; i++) {
+            mbh.get().put((byte) rand.nextInt(255));
```

Comment:
Could it be possible to check the content of buffer using java.util.zip.CRC32 ? and the result would be shown in testOutput().